tests/common/mellanox_data.py

```diff
@@ -559,6 +559,59 @@
                 "number": 1
             }
         }
+    },
+    "ACS-MSN4410": {
```

Comment:
Why use hwsku instead of platform like the others here?

Reply:
Fixed, thank you.